src/daemon/daemon.cpp

```diff
@@ -621,7 +621,7 @@ try // clang-format on
     vm->start();
     vm->wait_until_ssh_up(std::chrono::minutes(5));
 
-    reply.set_create_message("Waiting for cloud-init to complete");
+    reply.set_create_message("Waiting for boot and initialization to complete");
```

Comment:
It would probably be better to just say 'initialization' here since the VM is already booted at this point.